finder-app/finder.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+filesdir=$1
+searchstr=$2
+cd
+if [ "$#" -ne 2 ] ; then
+	echo "2 arguments required"
+	echo "1: the file directory path"
+	echo "2: the name of the file"
+	exit 1
+else	
+	if [ -d "${filesdir}" ] ; then
+		cd "${filesdir}"
+		numlines=$(sudo grep -x -r "${searchstr}" * | wc -l)
```

Comment:
@yogesh-1303 sudo shouldn't strictly be necessary here, this might be a problem later when we run on target because I don't think you'll have a sudo setup.